boot/core/src/boot/task_helpers/notify.clj

```diff
@@ -23,12 +23,18 @@
     (io/copy (io/input-stream (io/resource "boot-logo-3.png")) f)
     (.getAbsolutePath f)))
 
+(defn- warn-ex-thrown []
+  (util/warn
+   "An exception was thrown while trying to send a notification. To see more info, increase the verbosity of your build with e.g. \"boot -v\" or \"boot -vv\"\n"))
+
 (defn sh-with-timeout [& args]
   (try
     (apply util/dosh-timed 1000 args)
     0
     (catch Exception e
-      (util/print-ex e)
+      (if (= 1 @util/*verbosity*)
```

Comment:
Yeah, so that was my thought.  At `*verbosity*` level 1 `print-ex` will actually print the whole stacktrace, which is why I put this `if` in to only print a short warning message that an exception was thrown.  And then, if the `*verbosity*` is set higher or lower the appropriate thing will happen (i.e. at quiet `*verbosity*` nothing will be printed, but at higher levels the full exception will be shown.)